diff --git a/node_modules/cld/index.d.ts b/node_modules/cld/index.d.ts
index 2e5e5b9..cfaad9e 100644
--- a/node_modules/cld/index.d.ts
+++ b/node_modules/cld/index.d.ts
@@ -23,9 +23,9 @@ interface DetectLanguage {
   readonly languages: Language[];
   readonly chunks: Chunk[];
 }
-export declare module 'cld' {
-  declare function detect(text: string, options: Options, callback: (err: string, result: DetectLanguage) => void): void;
-  declare function detect(text: string, callback: (err: string, result: DetectLanguage) => void): void;
-  declare function detect(text: string, options: Options): Promise<DetectLanguage>;
-  declare function detect(text: string): Promise<DetectLanguage>;
+declare module 'cld' {
+  function detect(text: string, options: Options, callback: (err: string, result: DetectLanguage) => void): void;
+  function detect(text: string, callback: (err: string, result: DetectLanguage) => void): void;
+  function detect(text: string, options: Options): Promise<DetectLanguage>;
+  function detect(text: string): Promise<DetectLanguage>;
 }
